README.md

```diff
@@ -85,9 +82,9 @@ The entry creation might be affected by _visibility_: for instance, elements are
 
 #### What are some differences between text and image observation?
 
-* Whereas for images it is OK to care only about those which are associated to a resource, for text that is definitely not the case. An image not associated to a resource will need to be constructed manually and it is uncommon for such an image to be part of the key content of a website. On the other hand, text is relevant regardless of whether it is associated to a webfont or not. A webfont only makes it ‘pretty text’.
+* Whereas for images it is OK to care only about those which are associated to a resource, for text that is definitely not the case. An image not associated to a resource will need to be constructed manually and it is uncommon for such an image to be part of the key content of a website. Note that we consider inline images to have an associated resource, it is just inlined. On the other hand, text is relevant regardless of whether it is associated to a webfont or not.
```

Comment:
Is "inline" image a well defined concept?

Reply:
I think it's well understood? I could add "(those with data URIs)"

Reply:
I think that would be a bit clearer.